src/Symfony/Bridge/Doctrine/Tests/Validator/Constraints/UniqueEntityValidatorTest.php

```diff
@@ -574,6 +574,11 @@ public function testValidateUniquenessWithCompositeObjectNoToStringIdEntity()
 
         $objectOne = new SingleIntIdNoToStringEntity(1, 'foo');
         $objectTwo = new SingleIntIdNoToStringEntity(2, 'bar');
+
+        $this->em->persist($objectOne);
```

Comment:
Ah ok. I just checked the error:

```
\UniqueEntityValidatorTest::testValidateUniquenessWithCompositeObjectNoToStringIdEntity

Doctrine\ORM\ORMException: Entity of type Symfony\Bridge\Doctrine\Tests\Fixtures\CompositeObjectNoToStringIdEntity has identity through a foreign entity Symfony\Bridge\Doctrine\Tests\Fixtures\SingleIntIdNoToStringEntity, however this entity has no identity itself. You have to call EntityManager#persist() on the related entity and make sure that an identifier was generated before trying to persist 'Symfony\Bridge\Doctrine\Tests\Fixtures\CompositeObjectNoToStringIdEntity'
```

weird that it only fails on php 7.1 though? Is it using a different version of Docrine?